spec/services/users/delete_spec.rb

```diff
@@ -68,6 +68,14 @@
     end.to change(FieldTest::Membership, :count).by(-1)
   end
 
+  it "deletes reactions to the user" do
+    build(:vomit_reaction, reactable: user).save!
```

Comment:
It looks like the default `user` in the vomit_reaction factory wasn't a moderator so the `permissions` error (category is not valid if negative reaction from untrusted user) was being set.

I've since updated the factory so vomit reactions will be created from a moderator/trusted user account.